src/cases/containers/index.ts

```diff
@@ -5,7 +5,7 @@ import {CaseSearchComponent} from './case-search/case-search.component';
 import {CasesCreateComponent} from './case-create/case-create.component';
 import {CaseDetailsComponent} from './case-details/case-details.component';
 import { CaseCreateSubmitComponent } from './case-create-submit/case-create-submit.component';
-import {CaseDetailsHomeComponent} from './case-details-home/case-details-home';
+import {CaseDetailsHomeComponent} from './case-details-home/case-details-home.component';
```

Comment:
Inconsistent formatting. I could quickly change this?

Reply:
please do it, Thanks